ruby/neo4j/driver/config.rb

```diff
@@ -4,12 +4,15 @@ module Neo4j
   module Driver
     class Config < Hash
       include Ext::ConfigConverter
-      # class TrustStrategy
-      #   class << self
-      #     def trust_all_certificates; end
-      #   end
-      # end
-      TrustStrategy = Java::OrgNeo4jDriver::Config::TrustStrategy
+      class TrustStrategy
+        TRUST_ALL_CERTIFICATES = :trust_all_certificates
+        TRUST_CUSTOM_CA_SIGNED_CERTIFICATES = :trust_custom_ca_signed_certificates
+        TRUST_SYSTEM_CA_SIGNED_CERTIFICATES = :trust_system_ca_signed_certificates
+
+        def self.trust_all_certificates
+          TRUST_ALL_CERTIFICATES
+        end
+      end
```

Comment:
this class is incomplete. I would not replace the java class if you don't have a complete ruby replacement.

Reply:
@klobuczek yes, I am completing it along with it's dependent classes.